src/components/landing-page/call-to-action/CallToAction.jsx

```
@@ -1,22 +1,122 @@
+import {
+  CoursesCard,
+  SkillCards,
+  WhatOurUsersSay,
+} from "./Cards";
+import {
+  courses,
+  skillContent,
+  whatOurUsersSay,
+} from "./Content";
+import "./responsive.css";
+import WhyChooseAurora from "../why-choose-aurora";
```

ESLint

[error] 'WhyChooseAurora' is defined but never used.

(no-unused-vars)